collator.t3rn.Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:21.04
+LABEL org.opencontainers.image.source https://github.com/t3rn/t3rn
+
+ENV RELAYCHAIN_NAME=polkadot
+ENV PARACHAIN_NAME=t3rn
+
+RUN useradd -m -u 1000 -U -s /bin/sh -d /node runner && \
+    mkdir /node/data && \
+    mkdir /node/keystore && \
+    mkdir /node/specs
+
+COPY --chown=runner target/release/${PARACHAIN_NAME}-collator /usr/local/bin/${PARACHAIN_NAME}-collator
+COPY --chown=runner specs/$RELAYCHAIN_NAME.raw.json /node/specs/$RELAYCHAIN_NAME.raw.json
+COPY --chown=runner specs/$PARACHAIN_NAME.raw.json /node/specs/$PARACHAIN_NAME.raw.json
```

Comment:
We should also regen and check-in the t3rn specs under `./specs` ontop of the pure shell runtime tag.

Reply:
done